flake.nix

```diff
@@ -24,5 +24,21 @@
 
             doCheck = false;
           };
-  };
+
+      packages.x86_64-linux.dockerContainer = let
+          pkgs = import nixpkgs { system = "x86_64-linux"; };
+      in
+          pkgs.dockerTools.buildImage {
+            name = "centrifuge-chain";
+            tag = "latest";
```

Comment:
This should be the same version as the app, so I think 2.0?

Reply:
Feel free to define both name and version in a variable and reuse them across the file.